dts/x86/intel/ia32.dtsi

```diff
@@ -64,6 +64,7 @@
 			reg = <0xfed00000 0x400>;
 			interrupts = <2 IRQ_TYPE_FIXED_EDGE_RISING 4>;
 			interrupt-parent = <&intc>;
+			no-legacy-irq;
```

Comment:
This can likely also be added in the testsuite in a board overlay, maybe with a descriptive comment, in order to avoid breaking unrelated things.

Reply:
This will also go to the test suite